Gems/SceneProcessing/Code/Source/Generation/Components/MeshOptimizer/MeshOptimizerComponent.cpp

```diff
@@ -106,7 +106,7 @@ namespace AZ::SceneGenerationComponents
         auto* serializeContext = azrtti_cast<AZ::SerializeContext*>(context);
         if (serializeContext)
         {
-            serializeContext->Class<MeshOptimizerComponent, GenerationComponent>()->Version(3);
+            serializeContext->Class<MeshOptimizerComponent, GenerationComponent>()->Version(2);
```

Comment:
why does the version go down from 3 to 2? normally it goes up :)

Reply:
This is just because it is a pure commit revert (from a commit from an hour ago). Wanted to undo this as soon as possible.